path_config.go

```diff
@@ -24,8 +24,14 @@ func pathConfig(b *azureAuthBackend) *framework.Path {
 			},
 			"environment": {
 				Type:        framework.TypeString,
+				Deprecated:  true,
```

Comment:
my vote is to keep using this field and not deprecate it. I don't see much value in changing it.

Reply:
Reverted to `environment` in latest commit. Thanks for the callout!